examples/cavity/cavity.jl

```diff
@@ -259,7 +259,7 @@ function generate_cavity_convergence_data(;
         push!(dof, eltype(dof)())
         push!(f_TM_010, eltype(f_TM_010)())
         push!(f_TE_111, eltype(f_TE_111)())
-        for ref = ref_min:(p > 3 ? 3 : ref_max)
+        for ref = (p == 1 && mesh_type == 2 ? 1 : ref_min):(p > 3 ? 3 : ref_max)
```

Comment:
I think that should be 
`p == 1 && mesh_type == 2 ? max(1,refmin) : ref_min`

If a refmin of 2 was given for mesh_type 2, this would alway start from 1 as written. In fact the same fix should be applied on the max branch, something like `p > 3 : min(ref_max, 3) : ref_max`